### PR TITLE
AcceptOfferController/Statuses/issue 70

### PR DESCRIPTION
**Pull request by anish97IND**

### CODE REVIEW

File: src/main/java/com/kodekonveyor/work_request/offer/AcceptOfferController.java [hunk 1]
```diff
@@ -49,4 +50,11 @@ public WorkRequestDTO call(final long offerId) {
     workRequestDTO.setAddress(addressDTO);
     return workRequestDTO;
   }
+
+  private WorkRequestDTO
+      getWorkRequestDTOStatusAgreed(final WorkRequestEntity workRequest) {
+    final WorkRequestDTO workRequestDTO = getWorkRequestDTO(workRequest);
+    workRequestDTO.setStatus(WorkRequestStatusEnum.AGREED);
```
Comment (magwas) [inline src/main/java/com/kodekonveyor/work_request/offer/AcceptOfferController.java:57]: You set the status in the DTO, but not in the entity. The change is not persisted.

File: src/test/java/com/kodekonveyor/work_request/offer/AcceptOfferControllerStatusesTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  private WorkRequestDTO workRequestDTO;
+
+  @BeforeEach
+  public void setUpTest() {
+    workRequestDTOTestData = WorkRequestDTOTestData.getStatusAgreed();
+    workRequestDTO = acceptOfferController.call(OfferDTOTestData.get().getId());
+  }
+
+  @Test
+  @DisplayName("Work request status AGREED returned successfully")
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/offer/AcceptOfferControllerStatusesTest.java:38]: You should have a test checking that the work request is saved with the right status, and
you should check that the provider is set both in the saved workrequest and the returned dto

**Comment by magwas** [comment]
I have resolved the merge conflict for you. Please pull before push.

**Comment by anish97IND** [comment]
> I have resolved the merge conflict for you. Please pull before push.

Thanks Arpad. I guess some issues has come up regarding mutation coverage here.

**Pull request merged**